BE/todoList/src/main/java/com/codesquad/todoList/entity/Columns.java

```diff
@@ -0,0 +1,67 @@
+package com.codesquad.todoList.entity;
+
+import com.codesquad.todoList.error.exception.NotFoundCardException;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Column;
+import org.springframework.data.relational.core.mapping.Table;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Size;
+import java.util.*;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Table(value = "COLUMNS")
+public class Columns {
+
+    @Id
+    private Long id;
+
+    @NotBlank(message = "콜럼의 이름은 비어 있을 수 없습니다.")
+    @Size(max = 50, message = "콜럼의 이름은 최대 50자 미만입니다.")
+    private String name;
+
+    private List<Card> cardList = new ArrayList<>();
+
+    public Columns(String name) {
+        this.name = name;
+    }
+
+    public void addCard(Card card) {
+        cardList.add(card);
+    }
+
+    public Card deleteCard(String cardId) {
+        Card card = null;
+        for(Card originalCard : cardList) {
+            if(originalCard.getId().equals(cardId)) {
+                card = originalCard;
+                cardList.remove(originalCard);
+                return card;
+            }
+        }
+        throw new NotFoundCardException();
```

Comment:
포매팅 신경써주세요.
```suggestion
        Card card = cardList.stream().filter(card → card.equalsId(cardId)).findFirst().orElseThrow(NotFoundCardException::new);
        cardList.remove(card)
        return card;
```

정도로 stream을 활용해보면 되지 않을까요?

Reply:
그래도 Column이 Card를 포함한다는 느낌을 주려고 고생하셨네요~

Reply:
람다를 안쓰다보니 까먹고있었네요! 난잡한 부분들 수정하도록 하겠습니다. 😃